### ISSUE TITLE
Error accessing database during exchange query (kraken)

### ISSUE THREAD

**Issue by yabirgb** [rotki/rotki#5067]
rotki version: develop

logs

```
[31/10/2022 20:13:38 CET] ERROR rotkehlchen.greenlets Greenlet with id 140129837313536: Query history of Kraken exchange died with exception: 'NoneType' object has no attribute 'read_ctx'.
Exception Name: <class 'AttributeError'>
Exception Info: 'NoneType' object has no attribute 'read_ctx'
Traceback:
   File "src/gevent/greenlet.py", line 908, in gevent._gevent_cgreenlet.Greenlet.run
  File "/home/yabirgb/work/rotki/rotkehlchen/exchanges/exchange.py", line 489, in query_history_with_callbacks
    trades_history = self.query_trade_history(
  File "/home/yabirgb/work/rotki/rotkehlchen/utils/mixins/lockable.py", line 46, in wrapper
    result = f(wrappingobj, *args, **kwargs)
  File "/home/yabirgb/work/rotki/rotkehlchen/exchanges/exchange.py", line 267, in query_trade_history
    with self.db.conn.read_ctx() as cursor:
```

It happened to me with a db that I just pulled. What I did is create a new account syncing with the database in the rotki servers and let it query. I got the error shown previously. Tried to reproduce but It didn't show again

**Comment by LefterisJP** [comment]
yeah but as I said in the discord this makes no sense. Would really need more information to reproduce an issue like this

**Comment by LefterisJP** [comment]
Should be addressed by #5068
